src/stores/business.ts

```diff
@@ -269,54 +271,59 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
   const isAllowed = (action: AllowableActionE): boolean => {
     const isBusiness = !!currentBusiness.value?.identifier
 
-    const { isStaffAccount } = useBcrosAccount()
     const { getFeatureFlag } = useBcrosLaunchdarkly()
     const legalType = currentBusiness.value?.legalType
 
     switch (action) {
       case AllowableActionE.ADDRESS_CHANGE: {
         if (isEntityFirm()) {
-          return isAllowedToFile(FilingTypes.CHANGE_OF_REGISTRATION)
+          return isAllowedToFile(FilingTypes.CHANGE_OF_REGISTRATION) &&
+                 isAuthorized(AuthorizedActionsE.FIRM_CHANGE_FILING)
         }
-        return isAllowedToFile(FilingTypes.CHANGE_OF_ADDRESS)
+        return isAllowedToFile(FilingTypes.CHANGE_OF_ADDRESS) && isAuthorized(AuthorizedActionsE.ADDRESS_CHANGE_FILING)
```

Comment:
Are these checks in 2 places -- here (in isAllowed) and on the page where the buttons are rendered?

THIS code is specific to Allowable Actions, whereas we're working on Authorized Actions. I know they're similar but we need to keep them separate for now.

I don't think isAllowed should be looking at whether an action is authorized.